### DISCUSSION TITLE
Fatal Error - Unable to Activate after Installation

### DISCUSSION THREAD

**Discussion by reditechoz** [liedekef/events-made-easy#618] [General]
Have just installed the plugin on Wordpress 6.6.1

When I go to activate the plugin the following error message comes up:

Plugin could not be activated because it triggered a fatal error.
Parse error: syntax error, unexpected ':', expecting ')' in .../plugins/events-made-easy/eme-filters.php on line 252

It also mentions line 286 with same error.

Any ideas

**Comment by liedekef** [comment]
This means your php version is not updated. You're probably on php 7.x, while that has been out of support since the beginning of last year (see https://www.php.net/supported-versions.php ). Update to php 8.x and you'll be good to go.
